shopinvader/models/shopinvader_product.py

```diff
@@ -93,8 +93,8 @@ def _compute_shopinvader_category(self):
             for categ in categs:
                 parents = self.env["shopinvader.category"].search(
                     [
-                        ("parent_left", "<=", categ.parent_left),
-                        ("parent_right", ">=", categ.parent_right),
+                        ("record_id.parent_left", "<=", categ.parent_left),
```

Comment:
@rousseldenis I would prefer to use 'parent_of' operator

Reply:
Done.

I've added in another commit the same field of correction in wizard to bind categories.